src/main/java/duke/Event.java

```diff
@@ -0,0 +1,45 @@
+package duke;
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+
+/**
+ * Event class extends Task class as well and is a category of tasks that
+ * includes a description start date and end date.
+ */
+
+public class Event extends Task {
+
+    private LocalDate date1;
+    private LocalDate date2;
+
+    public Event(String name, boolean done, String date1, String date2) {
+        super(name, done);
+        this.date1 = LocalDate.parse(date1);
+        this.date2 = LocalDate.parse(date2);
+    }
+
+
+    public LocalDate getDate1() {
+        return this.date1;
+    }
+
+    public LocalDate getDate2() {
+        return this.date2;
+    }
+
+    @Override
+    public String summary() {
+        String s = "E___";
+        String d = this.getDone() ? "✓" : "X";
+        String p = this.getPriority() + "___";
+        return s + d + "___" + p + this.getTask() + "___" + this.getDate1() + "___" + this.getDate2();
```

Comment:
I like that you remember to leave a space between the operands and the operators, great readability!